hypertrace-trace-enricher/hypertrace-trace-enricher-impl/src/main/java/org/hypertrace/traceenricher/enrichment/EnrichmentProcessor.java

```diff
@@ -22,15 +32,34 @@
   private static final String ENRICHMENT_ARRIVAL_TIME = "enrichment.arrival.time";
   private static final Timer enrichmentArrivalTimer =
       PlatformMetricsRegistry.registerTimer(DataflowMetricUtils.ARRIVAL_LAG, new HashMap<>());
-  private final List<Enricher> enrichers = new ArrayList<>();
+
+  // Must use linked hashmap
+  private final Map<String, Enricher> enrichers = new LinkedHashMap<>();
+
+  private static final String ENRICHED_TRACES_COUNTER = "hypertrace.enriched.traces";
```

Comment:
nit: We can get rid of the `hypertrace.` prefix

Reply:
This is the metric convention we are using across hypertrace projects.
Also, this may cause compatibility issue if we change the name and existing dashboards may be broken.

@kotharironak @rish691: please confirm if changing and simplifying this metric name is okay?

Reply:
It is exposed in ready to use dashboard - https://github.com/hypertrace/hypertrace/tree/main/kubernetes/monitoring/dashboards. So would suggest keeping it as is now. And, we can clean up if we decided to go with `metrics.reporter` based prefix only in application.conf